### ISSUE TITLE
Bug: When adding new component variation code needs to be formatted in one specific way

### ISSUE THREAD

**Issue by akinorew** [SoCreate/angular-playground#112]
### Versions
```
angular-playground version 4.0.0
```

```
Output from: `ng --version`.
Angular CLI: 1.7.3
Node: 9.0.0
OS: darwin x64
Angular: 5.2.8
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router, upgrade

@angular/cdk: 5.2.4
@angular/cli: 1.7.3
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.2
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.6.2
webpack: 3.11.0

```

### Repro steps
* Adding next component variation without the new line doesn't add that option in the menu:

`.add('big', {
        template: `<div></div>`
    }).add('with specified icon', {
        template: `<div></div>`
    });`

* Only this code will work:

` .add('big', {
        template: `<div></div>`
    }) //there has to be new line here

.add('with specified icon', {
        template: `<div></div>`
    });`

### Desired Behavior
Both should work

**Comment by davidwuerfel** [comment]
@akinorew Could you check whether the PR would also fix your problem? My problem was very similar.
https://github.com/SoCreate/angular-playground/pull/109

**Comment by akinorew** [comment]
@davidwuerfel It doesn't fix my case, but this one would:
const scenarioRegex = /^.*(?!\/\/)\s*\.add\s*\(\s*['"](.+)['"]\s*,\s*{/gm;

**Comment by davidwuerfel** [comment]
@akinorew I had a look at the current master (where PR #109 has been integrated). It should also match your case. The Regex you provided would unfortunately break the "commented-out" case (there also exists a test for this). Could you check it again e.g. here: https://regex101.com/r/mtp2Fy/3 (I added your case)? If it is fixed, you may use the master or wait for a new patched version (@lurock: Are there any plans for a next official version?).

btw: I'm not part of the team, just had a very similar problem and try to help.

**Comment by akinorew** [comment]
@davidwuerfel Yes, you are absolutely right! It will fix my case too. I figured you are not part of the team, but thank you for responding!

**Comment by davidwuerfel** [comment]
@akinorew Glad to hear. You're welcome.

**Comment by lurock** [comment]
This should now be fixed in the 4.0.1 release that is now available on npm.
